### PR TITLE
Jt 56

### PR DESCRIPTION
**Pull request by saffchen**
Change the build system of the project

### CODE REVIEW

File: buildSrc/build/generated-sources/kotlin-dsl-accessors/kotlin/gradle/kotlin/dsl/accessors/_fa8ae7a11e714e4557697295cd0bbd61/Accessors15c80wxji7qdhvcsteytc58m6.kt [hunk 1]
```diff
@@ -0,0 +1,68 @@
+
```
Comment (pelmegov) [inline buildSrc/build/generated-sources/kotlin-dsl-accessors/kotlin/gradle/kotlin/dsl/accessors/_fa8ae7a11e714e4557697295cd0bbd61/Accessors15c80wxji7qdhvcsteytc58m6.kt:1]: Is everything under .gradle folder required in git?

Comment (pelmegov) [reply buildSrc/build/generated-sources/kotlin-dsl-accessors/kotlin/gradle/kotlin/dsl/accessors/_fa8ae7a11e714e4557697295cd0bbd61/Accessors15c80wxji7qdhvcsteytc58m6.kt:1]: Same question about .mvn

Comment (saffchen) [reply buildSrc/build/generated-sources/kotlin-dsl-accessors/kotlin/gradle/kotlin/dsl/accessors/_fa8ae7a11e714e4557697295cd0bbd61/Accessors15c80wxji7qdhvcsteytc58m6.kt:1]: Fixed. This folder has been add to git ignore.

Comment (saffchen) [reply buildSrc/build/generated-sources/kotlin-dsl-accessors/kotlin/gradle/kotlin/dsl/accessors/_fa8ae7a11e714e4557697295cd0bbd61/Accessors15c80wxji7qdhvcsteytc58m6.kt]: Fixed

**Review by pelmegov — APPROVED** [review]

**Pull request merged**